config/genesis.go

```diff
@@ -64,22 +64,22 @@ func NewGenUserAccount(id, balance, tokenType, nonce, status uint64, publicKey s
 }
 
 type GenesisAccounts struct {
```

Comment:
Should be `GenesisStates`?
```suggestion
type GenesisStates struct {
```